tests/functional/analyze_and_parse/test_files/tidy_check.output

```diff
@@ -21,5 +21,5 @@ tidy_check.cpp:5:5: found assert() with side effect [misc-assert-side-effect]
     assert(++i);
     ^
 
-clang-tidy found 1 defect(s) while analyzing tidy_check.cpp
+Found 1 defect(s) while analyzing tidy_check.cpp
```

Comment:
Maybe it would be great to have a test case where both sa and tidy finds defects to see that the numbers are added up correctly.